nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/SdkPackageUpdaterTests.cs

```diff
@@ -29,277 +34,290 @@ public class SdkPackageUpdaterTests
-                      <ItemGroup>
-                        <PackageReference Include="AWSSDK.S3" Version="3.3.17.3" />
-                        <PackageReference Include="AWSSDK.Core" Version="3.3.21.19" />
-                      </ItemGroup>
-                    </Project>
-                    """),
+                (
+                    Path: "src/Project/Project.csproj",
+                    Content: """
+                             <Project Sdk="Microsoft.NET.Sdk">
+                               <PropertyGroup>
+                                 <TargetFramework>netstandard2.0</TargetFramework>
+                               </PropertyGroup>
+                               <ItemGroup>
+                                 <PackageReference Include="AWSSDK.S3" Version="3.3.17.3" />
+                                 <PackageReference Include="AWSSDK.Core" Version="3.3.21.19" />
+                               </ItemGroup>
+                             </Project>
+                             """),
             }, // starting contents
             new[]
             {
                 // If a dependency has a version constraint outside of our new-version, we don't update anything
-                (Path: "src/Project/Project.csproj", Content: """
-                    <Project Sdk="Microsoft.NET.Sdk">
-                      <PropertyGroup>
-                        <TargetFramework>netstandard2.0</TargetFramework>
-                      </PropertyGroup>
-                      <ItemGroup>
-                        <PackageReference Include="AWSSDK.S3" Version="3.3.17.3" />
-                        <PackageReference Include="AWSSDK.Core" Version="3.3.21.19" />
-                      </ItemGroup>
-                    </Project>
-                    """),
+                (
+                    Path: "src/Project/Project.csproj",
+                    Content: """
+                             <Project Sdk="Microsoft.NET.Sdk">
+                               <PropertyGroup>
+                                 <TargetFramework>netstandard2.0</TargetFramework>
+                               </PropertyGroup>
+                               <ItemGroup>
+                                 <PackageReference Include="AWSSDK.S3" Version="3.3.17.3" />
+                                 <PackageReference Include="AWSSDK.Core" Version="3.3.21.19" />
+                               </ItemGroup>
+                             </Project>
+                             """),
             }, // expected contents
-            "AWSSDK.Core",
-            "3.3.21.19",
-            "3.7.300.20",
-            false // isTransitive
+            new DependencyRequest { Name = "AWSSDK.Core", PreviousVersion = "3.3.21.19", NewVersion = "3.7.300.20", IsTransitive = false }
         ];
 
         // Dependency project has version constraint
         yield return
         [
             new[]
             {
-                (Path: "src/Project2/Project2.csproj", Content: """
```

Comment:
Could you revert the formatting/indentation changes in this and other files?  It makes it harder to see what's being changed.

Reply:
Is it good enough now?

Reply:
Yes, thank you, I'll keep looking/testing.